src/win32/Watcher.cpp

```diff
@@ -104,22 +104,32 @@ std::string getUTF8FileName(std::wstring path) {
   return utf8Directory;
 }
 
-Watcher::Watcher(std::shared_ptr<EventQueue> queue, HANDLE dirHandle, const std::wstring &path, bool pathWasNtPrefixed)
+Watcher::Watcher(std::shared_ptr<EventQueue> queue, const std::wstring &path, bool pathWasNtPrefixed)
   : mRunning(false),
-  mDirectoryHandle(dirHandle),
   mQueue(queue),
   mPath(path),
   mPathWasNtPrefixed(pathWasNtPrefixed)
 {
-  ZeroMemory(&mOverlapped, sizeof(OVERLAPPED));
-  mOverlapped.hEvent = this;
-  resizeBuffers(1024 * 1024);
-  mWatchedPath = getWatchedPath();
-  start();
+  std::this_thread::sleep_for(std::chrono::milliseconds(1000));
```

Comment:
I'm assuming this was for testing and can be removed?

Reply:
Yeah, I forgot to remove it